draft-irtf-cfrg-frost.md

```diff
@@ -1473,8 +1473,7 @@ The function `polynomial_interpolate_constant` is defined as follows.
 
     f_zero = Scalar(0)
     for (x, y) in points:
-      delta = y * derive_interpolating_value(x, x_coords)
-      f_zero = f_zero + delta
+      f_zero += y * derive_interpolating_value(x, x_coords)
```

Comment:
The other changes look good, but please revert this change. I think it's less clear without much benefit (saving one line).

Reply:
I changed this to be in line with the code for `polynomial_evaluate`, which also uses similar syntax. 

```
for coeff in reverse(coeffs):
      value *= x
      value += coeff
```

Reply:
The difference, to me, is that this one line now does two things: it multiplies two values, and then adds the result to the accumulator. The examples in `polynomial_evaluate` just do one thing.

Reply:
updated now.